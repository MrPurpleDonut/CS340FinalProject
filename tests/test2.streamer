text myStr = "This is line one with \"quotes inside\". \n This is line two.";
print myStr;
