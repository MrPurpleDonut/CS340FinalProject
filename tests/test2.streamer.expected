Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: myStr
            Type: text
            Assignment statement: myStr
                Quoted string: "This is line one with \"quotes inside\". \n This is line two."
        Print Statement
            Identifier: myStr
